sys/net/crosslayer/ng_pktbuf/_pktbuf_static.c

```diff
@@ -165,7 +168,7 @@ void *_pktbuf_internal_alloc(size_t size)
 
     while ((node->next != NULL)
            /* and if space between current and next allocation is not big enough */
-           && ((_start_idx(node->next) - _end_idx(node)) < _total_sz(size))) {
+           && ((_start_idx(node->next) - _end_idx(node)) < _al_sz(_total_sz(size)))) {
```

Comment:
Obviously I overlooked `node->next` and `node` in my error-searching-mania. Sorry! Second stupid question: (Why) is the `node->next != NULL` condition needed? Besides looking at the code I just wonder if there is a situation where you need to create a new node, because  the free memory is too small. How/where do you handle this?